sample/testRecorder-complete/testRecorder.spec.js

```diff
@@ -0,0 +1,27 @@
+require('./pages/home');
+require('./pages/product');
+
+describe("testRecorder", function () {
+
+  it("should validate the home screen", function () {
+    // Assertions
+    Then.onTheHomePage.iShouldSeeAllCategories();
+  });
+
+  it("Should search for a product", function () {
+    // Actions
+    When.onTheHomePage.iSearchForProduct();
+
+	// Assertions
+    Then.onTheHomePage.theProductListShouldBeFiltered();
+  });
+
+  it("Should navigate to the product", function () {
+    // Actions
+    When.onTheHomePage.iSelectTheFirstProduct();
```

Comment:
the search finds no products
instead, select the flat screen category and then select the first product